src/test/java/org/pesmypetcare/webservice/dao/usermanager/UserDaoTest.java

```diff
@@ -95,50 +124,39 @@ public static void setUp() {
     }
 
     @Test
-    public void shouldCreateUserAccount() throws FirebaseAuthException {
-        dao.createUserAuth(userEntity, password);
-        verify(myAuth).createUser(isA(UserRecord.CreateRequest.class));
-    }
-
-    @Test
-    public void shouldThrowFirebaseAuthExceptionOnFail() {
-        assertThrows(FirebaseAuthException.class, () -> {
-            willThrow(FirebaseAuthException.class).given(myAuth).createUser(any(UserRecord.CreateRequest.class));
-            dao.createUserAuth(userEntity, password);
-        }, "Should throw FirebaseAuthException when the creation fails");
-    }
-
-    @Test
-    public void shouldCreateUserOnDatabase() throws DatabaseAccessException, FirebaseAuthException,
-        ExecutionException, InterruptedException {
+    public void shouldCreateUserOnDatabase()
+        throws DatabaseAccessException, FirebaseAuthException, ExecutionException, InterruptedException {
         given(usedUsernames.document(anyString())).willReturn(usernameRef);
         given(users.document(anyString())).willReturn(userRef);
         given(usernameRef.get()).willReturn(future);
         given(future.get()).willReturn(snapshot);
         given(snapshot.exists()).willReturn(false);
-        given(usernameRef.set(docData)).willReturn(null);
-        given(userRef.set(any(UserEntity.class))).willReturn(null);
         given(myAuth.getUser(uid)).willReturn(userRecord);
         given(userRecord.updateRequest()).willReturn(updateRequest);
         given(updateRequest.setDisplayName(anyString())).willReturn(updateRequest);
+        given(db.batch()).willReturn(batch);
+        given(batch.commit()).willReturn(batchResult);
+        given(batchResult.get()).willReturn(null);
 
         dao.createUser(uid, userEntity);
         verify(usedUsernames, times(2)).document(same(username));
-        verify(usernameRef).set(docData);
+        verify(batch).set(usernameRef, docData);
         verify(users).document(same(uid));
-        verify(userRef).set(same(userEntity));
+        verify(batch).set(userRef, userEntity);
         verify(myAuth).getUser(same(uid));
         verify(updateRequest).setDisplayName(same(username));
+        verify(batch).commit();
     }
 
     @Test
-    public void shouldDeleteUser() throws DatabaseAccessException, FirebaseAuthException,
-        ExecutionException, InterruptedException, DocumentException {
+    public void shouldDeleteUser()
+        throws DatabaseAccessException, FirebaseAuthException, ExecutionException, InterruptedException, DocumentException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 124).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5492924)